app/controllers/people_controller.rb

```diff
@@ -88,6 +88,8 @@ def new
     @page_title = "New Person"
     @person = Person.new(status: cookies[:status], state: 'MA')
     @person.channels.build (attributes = {category: 'E-Mail', status: 'OK', usage: '1-All'})
+    @person.phones.build
+    @person.emails.build
     @mobile = @person.channels.build (attributes = {category: 'Mobile Phone', status: 'OK', usage: '1-All'})
```

Comment:
I would think we don't need both this and line 92. These lines are to build objects to be populated from the form. Also, these should only be on the new action, not the edit.